app/src/test/java/org/simple/clinic/editpatient/EditPatientEffectHandlerTest.kt

```diff
@@ -304,4 +304,17 @@ class EditPatientEffectHandlerTest {
     verify(ui).setupUi(inputFields)
     verifyNoMoreInteractions(ui)
   }
+
+  @Test
+  fun `when fetch colony or village names effect is received, then fetch them from the database`() {
+    // given
+    val colonyOrVillages = listOf("Colony1, Colony2, Colony3, Colony 4")
```

Comment:
I think you accidentally added the `,` in the String. Here it's a list of single string, it should be `listof("Colony1", "Colony2", "Colony3", "Colony4")`